bitnami/milvus/templates/_helpers.tpl

```diff
@@ -960,6 +967,8 @@ Init container definition for waiting for the database to be ready
     - -ec
     - |
       #!/bin/bash
+      # Copy logging configuration
+      cp /bitnami/milvus/conf/log/glog.conf /bitnami/milvus/rendered-conf/glog.conf
```

Comment:
Shouldn't this be copied to the log-config volume?

Reply:
The `log-config` volume is a configmap where the `glog.conf` is provided. Hence we need to copy it to `/opt/bitnami/milvus/configs`, but this done via `prepare-mivlus` init container. 

The init container `prepare-milvus` mounts the volume `rendered-config` on ` /bitnami/milvus/rendered-conf/`, this same volume is mounted later by the `milvus` container on `/opt/bitnami/milvus/configs`.